src/frontend/Komponenter/Personoversikt/Klage/OpprettKlage.tsx

```diff
@@ -54,17 +55,22 @@ export const OpprettKlage: React.FunctionComponent<IProps> = ({
     fagsak,
     settVisModal,
     opprettKlage,
+    kanOppretteKlagebehandling,
 }) => {
     const [feilmelding, settFeilmelding] = useState<string>('');
     const [valgtDato, settValgtDato] = useState<string>();
 
     const sisteFerdigstilteBehandlingen = sisteBehandlingSomErFerdigstilt(fagsak);
 
     if (!sisteFerdigstilteBehandlingen) {
+        return <Normaltekst>Finnes ikke noen ferdigstilt behandling å klage på</Normaltekst>;
+    }
+    if (!kanOppretteKlagebehandling) {
         return (
-            <>
-                <Normaltekst>Finnes ikke noen ferdigstilt behandling å klage på</Normaltekst>
-            </>
+            <Normaltekst>
```

Comment:
```suggestion
            <BodyShort>
```

Dersom du ikke bruker `Alert`